.ci/scripts/setup-arm-baremetal-tools.sh

```diff
@@ -9,3 +9,5 @@
 git config --global user.email "github_executorch@arm.com"
 git config --global user.name "Github Executorch"
 bash examples/arm/setup.sh --i-agree-to-the-contained-eula
+
+./examples/models/llama3_2_vision/install_requirements.sh
```

Comment:
is this the best place to put this?

Reply:
The idea was to have it in place before the testing but thinking about it now I think you are right 
Having it as part of all the steps that use pytest in the file
`backends/arm/test/test_arm_baremetal.sh`
(or create a prepare test step)
Might be better and easier for anyone to just run the test and they will install needed dependencies them self.

Reply:
Done